ctk/common/src/main/kotlin/org/codice/compliance/verification/core/ResponseVerifier.kt

```diff
@@ -43,10 +44,10 @@ abstract class ResponseVerifier(private val samlRequest: RequestAbstractType,
 
     /** 3.2.2 Complex Type StatusResponseType */
     override fun verify() {
-        super.verify()
         verifyStatusResponseType()
         verifyStatusType()
         verifyStatusMessage()
+        super.verify()
```

Comment:
I did that because we have to validate the types before we do any common verification. For instance, if we don't run the string verification on version (which makes sure it's not blank) and run the SAML Versioning verifier. We get misleading errors.

Reply:
We could do it for all verifiers

Reply:
I think we should only do this if there are specific fields that we are calling CommonDataTypeVerifier on. For example, for `CoreAuthnRequestProtocolVerifier`, we need to call super first cause that's where the data types are verified.